lymph/unilateral.py

```diff
@@ -127,27 +128,40 @@ def init_edges(self, graph):
         self.tumor_edges = []
         self.lnl_edges = []
         self.growth_edges = []
-
-        for (_, start_name), end_name in graph.items():
-            new_edge = Edge(
-                start=self.find_node(start_name),
-                end=self.find_node(end_name),
-            )
-            if new_edge.is_tumor_spread:
-                self.tumor_edges.append(new_edge)
-            elif new_edge.is_growth:
+        for (typ, start_name), end_names in graph.items():
+            if typ == 'lnl':
+                new_edge = Edge(
+                    start=self.find_node(start_name),
+                    end=self.find_node(start_name),
+                )
                 self.growth_edges.append(new_edge)
-            else:
-                self.lnl_edges.append(new_edge)
+            for end_name in end_names:
+                new_edge = Edge(
+                    start=self.find_node(start_name),
+                    end=self.find_node(end_name),
+                )
+                if new_edge.is_tumor_spread:
+                    self.tumor_edges.append(new_edge)
+                else:
+                    self.lnl_edges.append(new_edge)
 
 
     def __str__(self) -> str:
         """Print info about the instance."""
         return f"Unilateral with {len(self.tumors)} tumors and {len(self.lnls)} LNLs"
 
+
     def print_info(self):
         """Print detailed information about the instance."""
-        raise NotImplementedError("Not yet implemented!")
+        num_tumors = len(self.tumors)
+        num_lnls   = len(self.lnls)
+        string = (
+            f"Unilateral lymphatic system with {num_tumors} tumor(s) "
+            f"and {num_lnls} LNL(s).\n"
+            + " ".join([f"{e} {e.spread_prob}%" for e in self.tumor_edges]) + "\n" + " ".join([f"{e} {e.spread_prob}%" for e in self.lnl_edges])
+            + f"\n the growth probability is: {self.growth_edges[0].spread_prob}" + f" the micro mod is {self.lnl_edges[0].micro_mod}"
+        )
+        print(string)

```

Comment:
Eventually, I thought about writing a method like this to generate a [Mermaid graph](https://mermaid.js.org/). But anything works for now.

Reply:
Yes, we definitely need a better solution. This is just a temporal fix to get some info